08-Feb-24 01:33:50 | root - INFO - Start getting MTR Bus stops
08-Feb-24 01:33:50 | root - ERROR - Other error occurred: list index out of range
08-Feb-24 01:33:50 | root - ERROR - list index out of range
Traceback (most recent call last):
  File "/home/runner/work/BusData/BusData/action/MTR_BUS_Route.py", line 91, in main
    stop['bound'] = row[1]
                    ~~~^^^
IndexError: list index out of range
